server/src/controller/error.js

```diff
@@ -0,0 +1,23 @@
+const clientError = (req, res) => {
+  res.status(404).json({ error: 'Page Not Found' });
+};
+
+// eslint-disable-next-line no-unused-vars
+const serverError = (err, req, res, next) => {
+  // eslint-disable-next-line no-console
+  console.log(err);
+  if (err.isBoom) {
+    // boom error
+    const {
+      output: { statusCode, payload },
+    } = err;
+    res.status(statusCode).send(payload);
+  } else {
+    res.status(err.status).json({ msg: err.msg });
```

Comment:
```suggestion
    res.status(500).json({ msg: "Something went wrong });
```
errors at the server side should not be displayed to clients